src/nlp/arrow_dataset.py

```diff
@@ -1325,91 +1468,11 @@ def shuffle(
         return self.select(
             indices=permutation,
             keep_in_memory=keep_in_memory,
-            load_from_cache_file=load_from_cache_file,
-            cache_file_name=cache_file_name,
+            indices_cache_file_name=indices_cache_file_name,
             writer_batch_size=writer_batch_size,
             verbose=verbose,
         )
 
-    def export(
```

Comment:
Moved this method without modification to keep all the samples re-ordering/selection methods (`select`, `sort`, `shuffle`, `shard`, `train_test_split`) in the same part of the file. Sorry for that.